src/NeuroPlots.jl

```diff
@@ -1,5 +1,171 @@
+
+    # Draw the circle around the head
+    poly!(
+        ax1,
+        Circle(Point2f(0, 0), maxX),
+        color = :transparent,
+        strokecolor = :black,
+        strokewidth = 4,
+    )
+
+    # Draw the nose
+    lines!(ax1, nosePoints(-10, maxX), color = :black, linewidth = 4)
+    lines!(ax1, nosePoints(+10, maxX), color = :black, linewidth = 4)
+
+    # Draw the ears
+    lines!(ax1, earPoints(-0.75, maxX), color = :black, linewidth = 4)
+    lines!(ax1, earPoints(+0.75, maxX), color = :black, linewidth = 4)
+
+    # Add Colorbar
+    Colorbar(gb[1, 1], width = 20, ticks = 0:0.1:1)
+
+    # Some decorations
+    colsize!(fig.layout, 1, Aspect(1, 1.0))
+
+    save("figure.png", fig, px_per_unit = 1) # output size = 1300 x 1000 pixels
+
+    return fig
 end
+
+"""
+    append_nearest_values(Xs, Ys, Vals; n = 8, radius = 1.2, k = 4)
+
+Append `n = 8` extra points to the original data at a circle with `radius = 1.2`. 
+The values of these points are calculated from the nearst `k = 4` points' values
+`Vals` at the original coordinates `(Xs, Ys)`. 
+"""
+function append_nearest_values(Xs, Ys, Vals; n = 8, radius = 1.2, k = 4)
+    newPoints = Array{Float64,2}(undef, n, 3)
+    for i = 1:n
+        coordinates = radius * exp(i * 2pi * im / n) # Euler's identity
+        distances = coordinates .- complex.(Xs, Ys)
+        newVal = mean(Vals[sortperm(abs.(distances)).<= k])
+        newPoints[i, :] = [real(coordinates), imag(coordinates), newVal]
+    end
+    return vcat([Xs Ys Vals], newPoints)
+end
+
+"""
+    nosePoints(angle, radius)
+
+Generate two points to define the left (`angle > 0`) or right (`angle < 0`) line 
+to draw the contour of the avator nose, starting from the surface with `radius`.
+"""
+nosePoints(angle, radius) =
+    [radius * Point2f(cosd(90 + angle), sind(90 + angle)), 0.85 * Point2f(0, 1)]
+
+"""
+    earPoints(focusX, radius; npoints = 100)
+
+Generate `npoints = 100` points to define the ellipse curve of the left (`FocusX < 0`) 
+or right (``FocusX > 0`) ear, with the focus point being at `(FocusX, 0)`.
+"""
+function earPoints(focusX, radius; npoints = 100)
+    coordinates = exp.(collect(1:npoints+1) * 2pi * im / npoints) # Euler's identity
+    modifyCoordinates =
+        complex.(focusX .+ 0.09 .* real(coordinates), 0.18 .* imag(coordinates))
+    modifyCoordinates[abs.(modifyCoordinates).< radius] .= NaN
+    return Point2f.(real(modifyCoordinates), imag(modifyCoordinates))
+end
+
+"""
+    my_channel
+
+Channel labels of the system I used
+"""
+const my_channel = String["Fpz", "Fp1", "Fp2", "AF3", "AF4", "AF7", "AF8", "Fz", "F1", "F2", "F3", "F4", "F5", "F6", "F7", "F8", "FCz", "FC1", "FC2", "FC3", "FC4", "FC5", "FC6", "FT7", "FT8", "Cz", "C1", "C2", "C3", "C4", "C5", "C6", "T7", "T8", "CP1", "CP2", "CP3", "CP4", "CP5", "CP6", "TP7", "TP8", "Pz", "P3", "P4", "P5", "P6", "P7", "P8", "POz", "PO3", "PO4", "PO5", "PO6", "PO7", "PO8", "Oz", "O1", "O2"];
```

Comment:
Is there a more meaningful name than `my_channel`? If this is just for the purpose of a quick example then we may want to just include it in documentation.